### DISCUSSION TITLE
Pay with different currencies

### DISCUSSION THREAD

**Discussion by GHVernova** [Paymenter/Paymenter#338] [Ideas]
Users can choose which currency to pay with

In the admin section we can activate if we want customers to pay with another currency and configure the available currencies we want them to pay with.

For example:
**Allow payment with different currencies: yes/no.**
-
**Currencies to pay with: EUR (€), USD ($)**

**Comment by GreenArrowlol** [comment]
multi currency was already suggested, please don't duplicate post.
